src/pixelator/pixeldataset.py

```diff
@@ -479,20 +570,25 @@ def from_file(path: PathType) -> PixelDataset:
     @staticmethod
     def from_data(
         adata: AnnData,
-        edgelist: pd.DataFrame,
+        edgelist: Optional[pd.DataFrame],
         metadata: Optional[Dict[str, Any]] = None,
         polarization: Optional[pd.DataFrame] = None,
         colocalization: Optional[pd.DataFrame] = None,
+        copy: bool = True,
+        allow_empty_edgelist: bool = False,
     ) -> PixelDataset:
         """Create a new instance of PixelDataset from the provided underlying objects.
 
         :param adata: an instance of `AnnData`
-        :param edgelist: an edgelist as a `pd.DataFrame`
+        :param edgelist: an edgelist as a `pd.DataFrame`, defaults to None
```

Comment:
It doesn't default to `None`. It is just optional

Reply:
Should we just delete the defaults to None if we need it not to?

Reply:
Yes. Good catch. I just missed setting this back since I went a bit back and forth in the implementation of this.